test/include.js

```diff
@@ -196,3 +196,33 @@ test('includes from opts.source throws', function t(assert) {
         });
     }
 });
+
+test('include non-relative filename', function t(assert) {
```

Comment:
Please also cover that absolute paths are forbidden. That will require an absolute path IDL text fixture if we don’t have one already.